lib.js

```diff
@@ -102,6 +102,8 @@ export const ensureDirs = () => {
 
   gitCheck(['config', 'user.name'], workDir, 'vrün\n', 'wrong user for work repository')
   gitCheck(['config', 'user.email'], workDir, 'db@vrün.com\n', 'wrong email for work repository')
+
+  console.debug('ensureDirs finished, all dirs are set up')
```

Comment:
or maybe these are desired :) that's fine if so.

Reply:
This one I do like to see, since it's only printed once as reassurance that all dirs are ok when starting the server